internal/pkg/manifest/workload.go

```diff
@@ -304,8 +305,8 @@ func UnmarshalWorkload(in []byte) (interface{}, error) {
 }
 
 func requiresBuild(image Image) (bool, error) {
-	hasBuild, hasURL := image.Build.isEmpty(), image.Location == nil
-	if hasBuild == hasURL {
+	noBuild, noURL := image.Build.isEmpty(), image.Location == nil
+	if noBuild == noURL {
```

Comment:
```suggestion
	if noBuild || noURL {
```

Reply:
i think it should still be xnor logic here. they just cannot be set together or not set for neither of them.

Reply:
oh oh i see you are also trying to catch the situation where both of them are set. I thought the error message was only for checking whether either one was set.

ok cool. Maybe we can add a comment on the side as this is an unusual if-statement:
```suggestion
	if noBuild == noURL { // err if both of them are specified or neither is specified
```